demo/index.html

```diff
@@ -25,6 +26,11 @@
         <button class="js-dialog-btn-close">Close dialog 1</button>
     </div>
 
+    <div id="my-dialog2" class="dialog js-dialog">
```

Comment:
Again, outside of this task and more just a general comment: even though these are demo files it'll be good to use the most semantic HTML so we're setting a good example. So in this case adding [`role="dialog"`](https://www.w3.org/TR/wai-aria-1.1/#dialog) as the `<dialog>` element still has hardly any support:

* http://caniuse.com/#feat=dialog
* http://www.html5accessibility.com/#controls -> Dialog

I see us providing decent documentation with each component where we'll have code examples.